src/helpers/sdks.js

```diff
@@ -97,4 +97,34 @@ module.exports = {
     }
     return Promise.resolve(['iOS SDK', 'N/A']);
   },
+
+  getWindowsSDKInfo: () => {
+    let info;
+    if (utils.isWindows) {
+      return utils
+        .run('reg query HKLM\\SOFTWARE\\Microsoft\\Windows\\CurrentVersion\\AppModelUnlock')
+        .then(out => {
+          const devModeLines = out
+            .split(/[\r\n]/g)
+            .slice(1)
+            .filter(x => x !== '');
+          const devModeMap = devModeLines.reduce((m, o) => {
+            const values = o.match(/[^\s]+/g);
+            m[values[0]] = values[2];
+            return m;
+          }, {});
+          info = devModeMap;
```

Comment:
why not just assign `info` to the result of your .reduce statement?

also assuming you aren't doing any error handling or logging here, you can just chain all of these.